actpool/actioniterator/actioniterator.go

```diff
@@ -45,17 +45,23 @@ type ActionIterator interface {
 }
 
 type actionIterator struct {
-	accountActs map[string][]action.SealedEnvelope
-	heads       actionByPrice
+	accountActs   map[string][]action.SealedEnvelope
```

Comment:
Instead of hacking the action itr, bp could bypass it, and also include the actions that it think needs to be processed (no matter system of user actions) into its proposed block